activities/recruitment/company-description.md

```diff
@@ -0,0 +1,43 @@
+---
+type: resource
+---
+
+# Company description for job advertising websites
+
+> Written initially for Stack Overflow (using their subheadings), this can be reused or remixed elsewhere.
+
+## About Foundation for Public Code
+
+We aim to build international public sector communities working together to share open source software and policy in a radically collaborative and open way. Public organizations are building more open software than ever before, but collaborating at the scale needed to co-develop and maintain a codebase is still hard - it's not the core responsibility of any single public organization.
+
+The Foundation for Public Code was set up to plug this infrastructure gap so that people already working for the public good can more easily:
+
+* Find transformative codebases to reuse
+* Design their code in a way that encourages international reuse
+* Build thriving open source communities around their codebases
+* Get institutional support in navigating the risk and procurement hurdles to using open source for public services and infrastructure
+
+We do this through active codebase stewardship and by publishing the [Standard for Public Code](https://standard.publiccode.net/).
+
+### How we work
+
+All of us fundamentally believe that our public organizations using FOSS will result in better public services for everyone and be a better use of taxpayer money. We do impactful work as part of a mission we believe in.
```

Comment:
I guess FOSS might work on Stack Overflow, but for other places we should perhaps not use the abbreviation unexplained.

Reply:
Thanks!